mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Modeling Intersections for Map Navigation
+---
+
+Capturing the various cases of Intersection modelling in OSM and the best practices to follow. The aim of this document is not go into the technicalities of traffic flow in a junction but rather to give a mapper clear cut instructions on how to model different intersections in OSM. Aiming to keep this document simple and easy to follow.
+
+### Case 1: When a transitioning dual carriageway intersects a bi-directional way. It has four ways in which the intersection can be designed 👇 
+
+![1(a)](https://user-images.githubusercontent.com/13744156/34767764-f09db5c2-f61e-11e7-8720-dc920acbb458.png)
+                                     *1(a)*
+
+![1(b)](https://user-images.githubusercontent.com/13744156/34767776-fbc70e80-f61e-11e7-8296-9d1a37d57548.png)
+                                     *1(b)*
+
+![1(c)](https://user-images.githubusercontent.com/13744156/34767825-1b2d1c42-f61f-11e7-8628-4ddb673fcc28.png)
+                                     *1(c)*
+
+So in case a dual carriageway intersects a bi-directional way to form a 4 way junction,1(b) variant is preferential for OSRM as it introduces less single-node intersections and for 1(c) additional turn restrictions are required to remove ambiguity in U-turns.
+
+For a transitioning divided way (dual carriageway) intersecting a single carriageway , this is the recommended model 👇 
+
+![image](https://user-images.githubusercontent.com/13744156/34767973-8b83f65a-f61f-11e7-8bec-f2f7e811634b.png)
+
+Note: 
+
+- Keep the geometry smooth. Don’t follow the centreline too rigorously as it might cause sharp turns
+
+
+### Case 2: When a transitioning dual carriageway intersects another dual carriageway, there are two methods to draw that intersection:
+
+#### Method 1:
+
+Collapse the dual carriageway after/before the intersection (Triangle after/before the intersection)
+
+
+![image](https://user-images.githubusercontent.com/13744156/34769147-048046fa-f623-11e7-889a-4977cf64d867.png)
+                                     *2(a)*
+
+Triangle before the intersection:
+
+![image](https://user-images.githubusercontent.com/13744156/34770557-79afb51a-f627-11e7-820e-ceccc5ebe4c7.png)
+                                      *2(b)*
+
+**Pros**: Helps in lane guidance as this parses the turn lane information much more accurately in OSRM.
```

Comment:
More accurately compared to what? Whether the mapper draws Boscell Road as one carriageway or two within the intersection, any segments within the intersection would need to be split anyways in order to remove right turns from the road’s `turn` indications.